Make.defs

```diff
@@ -21,27 +21,22 @@
 TOPDIR ?= $(APPDIR)/import
 include $(TOPDIR)/Make.defs
 
-# The GNU make CURDIR will always be a POSIX-like path with forward slashes
-# as path segment separators.  This is fine for the above inclusions but
-# will cause problems later for the native build.  If we know that this is
-# a native build, then we need to fix up the APPDIR path for subsequent
-# use
-
-ifeq ($(CONFIG_WINDOWS_NATIVE),y)
-APPDIR := ${shell echo %CD%}
-endif
-
 # Application Directories
 
 # BUILDIRS is the list of top-level directories containing Make.defs files
 # CLEANDIRS is the list of all top-level directories containing Makefiles.
 #   It is used only for cleaning.
 
-BUILDIRS   := $(dir $(wildcard $(APPDIR)$(DELIM)*$(DELIM)Make.defs))
-BUILDIRS   := $(filter-out $(APPDIR)$(DELIM)import$(DELIM),$(BUILDIRS))
-CONFIGDIRS := $(filter-out $(APPDIR)$(DELIM)builtin$(DELIM),$(BUILDIRS))
-CONFIGDIRS := $(filter-out $(dir $(wildcard $(APPDIR)$(DELIM)*$(DELIM)Kconfig)),$(CONFIGDIRS))
```

Comment:
@zouboan what's $(DELIM) on Windows native build environment? I think you can adjust $(DELIM) to the right value instead, which is more simpler than replacing it in many place.

Reply:
In Windows environment `DELIM := \` but` \ `has two role in Windows environment:
first: `\ `as directory separator, and second `\` as Escape character
in Windows environment:
`BUILDIRS   := $(dir $(wildcard $(APPDIR)$(DELIM)*$(DELIM)Make.defs))`
equivalent to
`BUILDIRS   := $(dir $(wildcard $(APPDIR)\*\Make.defs))`
and the `\` of `\*`  was identified to be Escape character, which result the `* ` was identified to be ordinary characters not used for pattern. and then result the BUILDIRS to be empty.
As described in: [https://www.gnu.org/software/make/manual/html_node/Text-Functions.html](url)
we can use double $(DELIM) to solve problem, and i succeed resolved a similar problem in f91c2383d67ff89f694e51bc59d11f226764be46 
but  when i use same method in apps/Make.defs , it give extra problem
@hartmannathan @xiaoxiang781216